web/src/lib/components/shared-components/side-bar/server-status.svelte

```diff
@@ -15,11 +15,14 @@
 
   let isOpen = false;
 
-  $: version = $serverVersion ? `v${$serverVersion.major}.${$serverVersion.minor}.${$serverVersion.patch}` : null;
-
   let info: ServerAboutResponseDto;
   let versions: ServerVersionHistoryResponseDto[] = [];
 
+  $: isMain = info?.sourceRef === 'main';
+  $: version = isMain
+    ? 'main ⚠️'
+    : $serverVersion && `v${$serverVersion.major}.${$serverVersion.minor}.${$serverVersion.patch}`;
+
```

Comment:
Feels kind of messy, suggestions welcome

Reply:
I think you want to keep the existing ternary so that it'll still be null. Not prettier though 😅